app/controllers/flows_controller.rb

```diff
@@ -622,22 +622,31 @@ def self.az_attributes(first_name: 'Testuser', last_name: 'Testuser')
         charitable_contributions: "yes",
         charitable_noncash: 123,
         contact_preference: "email",
-        current_step: "/en/questions/confirmation",
         eligibility_529_for_non_qual_expense: "no",
-        eligibility_lived_in_state: "yes",
         eligibility_married_filing_separately: "no",
-        eligibility_out_of_state_income: "no",
         email_address: "someone@example.com",
         email_address_verified_at: 1.minute.ago,
         has_prior_last_names: "yes",
         primary_first_name: first_name,
         primary_last_name: last_name,
         prior_last_names: "Jordan, Pippen, Rodman",
-        raw_direct_file_data: File.read(Rails.root.join('app', 'controllers', 'state_file', 'questions', 'df_return_sample.xml')),
         tribal_member: "yes",
         tribal_wages: 100,
-        payment_or_deposit_type: "mail",
-        account_type: 'unfilled'
+      )
+    end
+
+    def generate_efile_device_info(intake)
```

Comment:
this makes sure that we create the efile device infos when we generate an intake for the flow. Otherwise someone would have to go to the data review page and esign page in order to get the XML to work